test_scripts/Polices/Policy_Table_Update/021_ATF_P_TC_timeout_countdown_start.lua

```diff
@@ -0,0 +1,203 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- [PolicyTableUpdate] PoliciesManager changes status to “UPDATING”
+-- [HMI API] SystemRequest request/response
+--
+-- Description:
+-- SDL must forward OnSystemRequest(request_type=PROPRIETARY, url, appID) with encrypted PTS
+-- snapshot as a hybrid data to mobile application with <appID> value. "fileType" must be
+-- assigned as "JSON" in mobile app notification.
+-- 1. Used preconditions
+-- SDL is built with "-DEXTENDED_POLICY: EXTERNAL_PROPRIETARY" flag
+-- Application is registered.
+-- PTU is requested.
+-- SDL->HMI: SDL.OnStatusUpdate(UPDATE_NEEDED)
+-- SDL->HMI:SDL.PolicyUpdate(file, timeout, retry[])
+-- HMI -> SDL: SDL.GetURLs (<service>)
+-- HMI->SDL: BasicCommunication.OnSystemRequest ('url', requestType:PROPRIETARY, appID="default")
+-- SDL->app: OnSystemRequest ('url', requestType:PROPRIETARY, fileType="JSON", appID)
+-- 2. Performed steps
+--
+-- Expected result:
+-- SDL->HMI: SDL.OnStatusUpdate(UPDATING) right after SDL->app: OnSystemRequest
+---------------------------------------------------------------------------------------------
```

Comment:
Maybe need to add to expected result: 
`SDL waits for SystemRequest response from <app ID> within 'timeout' value, if no obtained, it starts retry sequence`